src/docker.go

```diff
@@ -29,87 +20,23 @@ var (
 
 func main() {
 	args := config.ArgumentList{}
-	i, err := integration.New(integrationName, integrationVersion, integration.Args(&args))
-	exitOnErr(err)
+	// we always use the docker API for OSes other than Linux
+	args.UseDockerAPI = util.UpdateDockerAPIArg(args.UseDockerAPI)
```

Comment:
We could also avoid modifying the `args` if possible. Maybe something like:

```go
useDockerAPI := util.shouldUseDockerAPI(args.UseDockerAPI)
```

We could also move the comment above to the OS specific implementation of the function.